### PR TITLE
Update home page with popovers

### LINKED ISSUE #59: [Task] Update Frontend index with Rover intro
- Finalize home page layout
- Include SmartRover intro from NASA data
- Apply nav menu styles

### PR DESCRIPTION
**Pull request by ramadi-a**
Update Frontend index with Rover intro
- Finalize home page layout
- Include SmartRover intro from NASA data
- Apply nav menu styles

Resolves #59

### CODE REVIEW

File: frontendApp/index.html [hunk 1]
```diff
@@ -1,39 +1,106 @@
...
+				</div>
+			</div>
+		</header>
+        <section class="rover-intro">
+			<div>
+				<img class="home-logo" src="assets/images/rover-logo.png" alt="SmartRover logo">
+				<img class="rover-model" src="assets/images/home-rover-model.png">
+			</div>
+			<div class="tooltip-row">
+				<div class="intro-tooltip1" data-bs-toggle="popover" data-bs-trigger="hover focus click" data-bs-placement="top" title="The Rover's &quot;Body&quot;" data-bs-html="true"
```
Comment (axe007) [inline frontendApp/index.html:34]: Please remove "click" trigger from data-bs-trigger. Popover is not hiding and persisting.

**Review by axe007 — CHANGES_REQUESTED** [review]

File: frontendApp/index.html (hunk 1 above)
Comment (ramadi-a) [reply frontendApp/index.html]: Removed "click" from line 34 in index.html

Comment (axe007) [reply frontendApp/index.html]: Works good now. Thanks.

**Review by axe007 — APPROVED** [review]
Good job. Now get busy with mission table please.

**Review by kam56 — COMMENTED** [review]
awesome. 👏  looks shiny and neat. Also, good that you included the comments in the .css file.

**Pull request merged**